workflow/main.jt

```diff
@@ -823,13 +845,61 @@ tools:
 
     input:
       is_allowed:
-        type: tring
+        type: string
       manifest_file:
         type: string
         is_file: true
       song_metadata_url:
         type: string
 
+  create_cgc_manifest:
+    command: |
+      bash -c '
```

Comment:
you may want to add `set -euxo pipefail`, see here: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/